packages/components/text/README.md

```diff
@@ -138,20 +139,21 @@ import Text from '@commercetools-uikit/text';
-| `isStrikethrough` | `Boolean`        |    -     | -                                                                             | `false` |                                                                                           |
-| `tone`            | `String`         |    -     | `['primary', 'secondary', 'information', 'positive', 'negative', 'warning'']` | -       |                                                                                           |
-| `children`        | `PropTypes.node` | ✅ (\*)  | -                                                                             | -       |                                                                                           |
-| `intlMessage`     | `intl message`   | ✅ (\*)  | -                                                                             | -       | An `intl` message object that will be rendered with `FormattedMessage`                    |
-| `title`           | `String`         |    -     | -                                                                             | -       |                                                                                           |
-| `truncate`        | `Bool`           |    -     | -                                                                             | `false` |                                                                                           |
-| `aria-labelledby` | `String`         |    -     | -                                                                             | -       | HTML ID of an element containing the label for the text rendered in this component        |
-| `nowrap`          | `Bool`           |    -     | -                                                                             | `false` | The content in the element will not be wrapped to a new line unless explicitly specified. |
+| Props             | Type             | Required | Values                                                                        | Default   |                                                                                           |
+| ----------------- | ---------------- | :------: | ----------------------------------------------------------------------------- | --------- | ----------------------------------------------------------------------------------------- |
+| `as`              | `string`         |    -     | `['small', 'span']` `[^]`                                                     | `false`   |                                                                                           |
+| `id`              | `String`         |    -     | -                                                                             | -         | Used as HTML id property                                                                  |
+| `isBold`          | `Boolean`        |    -     | -                                                                             | `false`   |
```

Comment:
We should also add a warning here about `isBold` is deprecated in favour of `fontWeight`.

Reply:
and also for the `Subheadline` component.

Reply:
@CarlosCortizasCT is it being deprecated in Subheadline? The ticket told me only to remove it for Body/Detail components.

Reply:
Oh! My bad. Sorry 🤦 

Forget about `Subhealine`.